src/components/atoms/ChefCard/ChefCard.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image"
+import Link from "next/link"
+
+type ChefCardProps = {
+  icon: string
+  name: string
+  text: string
+  args: string
+  count: number
+}
+
+export const ChefCard = (props: ChefCardProps) => {
+  return (
+    <Link href={props.args} className="flex gap-4">
+      <Image src={props.icon} width={88} height={116} alt="kitchenIcon" className="rounded-2xl" />
+      <div className="h-14 w-64 text-sm">
+        <h1 className="text-lg font-bold">{props.name}</h1>
+        <p className="my-1 line-clamp-3 h-14 text-Mauve-10">{props.text}</p>
```

Comment:
追加された`h-14`の影響で表示崩れてます〜！
<img width="356" alt="スクリーンショット 2023-07-22 18 38 19" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/67771199/93069b68-6984-433a-9f21-b25f4716e0f7">
```suggestion
        <p className="my-1 line-clamp-3 text-Mauve-10">{props.text}</p>
```

Reply:
ご指摘ありがとうございます🙏✨
こちらの確認してみると範囲内に収まっている感じなのですが…😭
田中さんから頂いた写真見ると思いっきりズレてますよね😢
一旦高さ消した方がよいでしょうか？

<img width="415" alt="スクリーンショット 2023-07-25 17 28 09" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/73697917/8f45dfd6-1390-42d4-8e3f-608522332251">

<img width="438" alt="スクリーンショット 2023-07-25 17 27 54" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/73697917/1e3f92c8-6445-4802-a33e-9c17f0a6157d">


<img width="369" alt="スクリーンショット 2023-07-25 17 28 02" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/73697917/24f80849-a256-4fb0-b8d6-9c645977db73">